source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -113,18 +113,22 @@ GLSSharpNavierStokesSolver<dim>::generate_cut_cells_map()
         {
           bool         cell_is_cut;
           bool         cell_is_inside;
-          unsigned int p_id_cut    = 0;
-          unsigned int p_id_inside = 0;
-          // is the particle index that cut the cell if it's cut.  If the cell
+          unsigned int p_id_cut                              = 0;
+          unsigned int p_id_inside                           = 0;
```

Comment:
use more verbose prefix than p_ please.
e.g. particle_id_which_cuts or something like that. Easier to understand for people reading your code.
<

Reply:
Perfect I applied this comment